cards/portfolio/addon/components/isolated.js

```diff
@@ -15,10 +18,17 @@ export default LiveIsolatedCard.extend({
   precision,
   selectedCurrency: service(),
   store: service(),
+  web3: service(),
+  metamaskBalances: A([]),
+  metamaskWallet: null,
 
   currency: readOnly('selectedCurrency.currency'),
   groupBy: 'wallets',
 
+  todaysRatesLookup: computed(function() {
+    return this.store.peekRecord('crypto-compare-current-rate', 'today');
```

Comment:
why are we doing this like this? this is already a relationships on the `portfolios` content-type: https://github.com/cardstack/portfolio/blob/508c013641356e191b038c4aaff2c9dba7d8027e/cards/asset/cardstack/static-model.js#L42